### PR TITLE
Add documentation for combined endpoints

### PR DESCRIPTION
**Pull request by StevenLangbroek**
I think this is dealt with nicely enough and the use-case is generic enough to warrant some extra documentation. Would love to hear thoughts on this!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,6 +229,77 @@ AppDispatcher.register((payload) => {
 });
 ```
 
+### Dealing with combined endpoints
+
+Sometimes, especially when dealing with low-bandwidth scenario's, you want to combine what would normally be multiple endpoints into one logical endpoint so that you save several server roundtrips. This works out of the box with normalizr. Let's say you want to get `articles` and `categories` to assemble the homepage of a blog.
```
Comment (devvmh) [inline README.md]: scenario's => scenarios

Comment (StevenLangbroek) [inline README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -229,6 +229,77 @@ AppDispatcher.register((payload) => {
...
+  entities: {
+    articles: {
+      1: {
+        id: 1,
+        title: 'Some Article'
+      }
+    },
+    categories: {
+      2: {
+        id: 1,
```
Comment (biesbjerg) [inline README.md]: Isn't this supposed to be id: 2? Sorry if I'm wrong, I'm still trying to wrap my head around it:)

**Comment by paularmstrong** [comment]
Thanks for the contribution! I agree that this is definitely something that can be made more explicit.

However, at this time, I'm working on #105 to completely rewrite the documentation. It's a bit hard to follow at the moment, so I'd like to get it all taken care of in one big rewrite than try to piece things together. I hope my closing this in favor of some bigger work doesn't discourage you from further PRs on normalizr.
